### PR TITLE
Replaces i4x examples with split examples

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2329](https://openedx.atlassian.net/browse/DOC-2329)

Updates examples in the B&R guides to use split course format rather than i4x.
### Date Needed

docathon!
### Reviewers
- [x] Subject matter expert: @JAAkana 
- [ ] Subject matter expert: @sstack22
- [x] Doc team review (sanity check): @mhoeber 

Please see https://docs.google.com/a/edx.org/document/d/1y_35d0IHvUrB--iL-BbC_mKqV5Q-No5m65wZK2gU9U4/edit?usp=sharing for before and after images.
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Update change log
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 1]
```diff
@@ -521,4 +523,4 @@ answer(s) can guide future changes to the courseware.
 
 
 .. _Using edX Insights: http://edx-insights.readthedocs.org/en/latest/
```
Comment (mhoeber) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst]: can we move to links file? (guess I missed these)

**Comment by mhoeber** [comment]
:+1: thanks

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 2]
```diff
@@ -518,7 +520,3 @@ toolbar and then select the type of chart you want.
 You can adjust your course content based on common student mistakes. While most
 students in this example selected the correct answer, the number of incorrect
 answer(s) can guide future changes to the courseware.
-
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst:521]: @mhoeber I fixed the Insights link, and it turned out the other one isn't even present.

**Comment by lamagnifica** [comment]
@JAAkana FYI if I were you I'd just do a find on the page with the files, to find the instances of "i4x" which should all be in pink, replaced by nice new examples in green... there's an automated test that runs on my commits now, and it removes extra spaces, resulting in a lot of unnecessary noise in these PRs...

File: en_us/olx/source/policies/course.rst [hunk 3]
```diff
@@ -76,15 +89,15 @@ Example Course Policy File
...
+	                   {"type": "course_info", "name": "Course Info"},
+	                   {"type": "discussion", "name": "Discussion"},
+	                   {"type": "wiki", "name": "Wiki"},
 	                   {"type": "progress", "name": "Progress"}],
-	
+
 	           "display_name": "edX Demonstration Course",
-	           "discussion_topics": {"General": {"id": "i4x-General-course-2014"}}
-               
+	           "discussion_topics": {"General": {"id": "course"}}
```
Comment (lamagnifica) [inline en_us/olx/source/policies/course.rst:99]: @sstack22 @jaakana ^

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 4]
```diff
@@ -89,8 +89,8 @@ to view.
 
   {
     "input_state": {
-      "i4x-edx-DemoX-problem-2363f9b08ed04be5a21e5a32511dd30a_2_1": {}
-    }, 
+      "e58b639b86db44ca89652b30ea566830_2_1": {}
+    },
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst:93]: @sstack22 @jaakana ^

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 5]
```diff
@@ -243,7 +245,7 @@ When you open the report, the value in the **State** column appears on a single
 line. This value is a record in JSON format. An example record for a text input
 capa problem follows.
 
-``{"correct_map": {"2363f9b08ed04be5a21e5a32511dd30a_2_1": {"hint": "", "hintmode": null, "correctness": "correct", "msg": "", "answervariable": null, "npoints": null, "queuestate": null}}, "input_state": {"2363f9b08ed04be5a21e5a32511dd30a_2_1": {}}, "last_submission_time": "2015-09-03T18:15:17Z", "attempts": 2, "seed": 1, "done": true, "student_answers": {"2363f9b08ed04be5a21e5a32511dd30a_2_1": "Nanjing University"}}``
+``{"correct_map": {"e58b639b86db44ca89652b30ea566830_2_1": {"hint": "", "hintmode": null, "correctness": "correct", "msg": "", "answervariable": null, "npoints": null, "queuestate": null}}, "input_state": {"e58b639b86db44ca89652b30ea566830_2_1": {}}, "last_submission_time": "2015-10-26T17:32:20Z", "attempts": 3, "seed": 1, "done": true, "student_answers": {"e58b639b86db44ca89652b30ea566830_2_1": "choice_2"}}``
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst:248]: @sstack22 @jaakana ^

File: en_us/shared/building_and_running_chapters/running_course/course_answers.rst [hunk 6]
```diff
@@ -284,7 +286,7 @@ see :ref:`Interpret a Student Submission History`.
 A **State** value that appears as follows indicates a learner who has viewed a
 capa problem, but not yet submitted an answer.
 
-  ``{"seed": 1, "input_state": {"i4x-edX-DemoX_1-problem-05c289c5ad3d47d48a77622c4a81ec36_2_1": {}}}``
+  ``{"seed": 1, "input_state": {"e58b639b86db44ca89652b30ea566830_2_1": {}}}``
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/course_answers.rst:289]: @sstack22 @jaakana ^

File: en_us/shared/building_and_running_chapters/running_course/discussions.rst [hunk 7]
```diff
@@ -124,35 +124,83 @@ To create a course-wide discussion topic, follow these steps.
 #. Scroll down to the **Discussion Topic Mapping** policy key. By default, its
    value looks like this.
 
- .. image:: ../../../shared/building_and_running_chapters/Images/Discussion_Add_initial.png
-  :alt: Policy value of {"General": {"id": "i4x-edX-Open-edx_demo_course"}}.
+   ::
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/discussions.rst]: @sstack22 @jaakana ^

File: en_us/shared/building_and_running_chapters/running_course/discussions.rst [hunk 8]
```diff
@@ -124,35 +124,83 @@ To create a course-wide discussion topic, follow these steps.
...
+
+     {
+         "General": {
+             "id": "course"
+         }
+     }
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/discussions.rst:133]: @sstack22 @jaakana ^

**Pull request merged**